projects/simple_mlp/train_with_pytorch_lightning.py

```diff
@@ -171,8 +171,8 @@ def main():
     output_dir.mkdir(exist_ok=True)
 
     # Load data
-    x = torch.load("../data/input_data.pt", weights_only=False)
-    y = torch.load("../data/labels.pt", weights_only=False)
+    x = torch.load("AI-ML-analytics-IE/projects/simple_mlp/data/input_data.pt", weights_only=False)

```

Comment:
remove the absolute paths. If you want to make it relative-path robust, you can use something like

os.path.abspath(os.path.join(os.path.dirname(__file__), "data", "input_data.pt"